content/en/tools/digital-currencies/defining/updating-token-definitions.md

```diff
@@ -0,0 +1,17 @@
+---
+date: '2023-03-21'
+lastmod: '2023-03-21'
+section_menu: tools
+draft: true
+menu:
+  tools:
+    name: "Updating Token Definitions"
+    weight: 600
+    parent: digital-currencies-token-definitions
+    identifier: digital-currencies-updating-token-definitions
+    description: "Digital Currencies documentation describing how to update existing token definitions via the GUI"
+title: "Updating Token Definitions"
+---
+
```

Comment:
```suggestion
## Updating Token Definitions

```

Reply:
Don't think we need an initial heading as that automatically comes from the Title of the page. Let me know if this is otherwise.